Gemfile

```diff
@@ -41,7 +41,7 @@ group :test, :development do
   gem 'factory_girl_rails'
   gem 'rspec-rails'
   gem 'pry-rails'
-  gem 'capybara-webkit'
+  #gem 'capybara-webkit'
```

Comment:
OK I will push a commit to this PR to re-add that to the gemfile. Thanks. Once we have a way to disable this feature (which I will work on today), I see no reason not to accept your PR.